restcord/invite_client.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+import logging
+from typing import Optional
+
+from aiohttp import ClientSession
+
+from .invite import Invite
+from .http import HTTPClient, Route
+
+__log__ = logging.getLogger(__name__)
+
+__all__ = (
+    'InviteClient'
+)
+
+class InviteClient(HTTPClient):
+
+    def __init__(self, token: str, loop=None, proxy=None, proxy_auth=None, session: Optional[ClientSession]=None) -> None:
+        super().__init__(token=token, loop=loop, proxy=proxy, proxy_auth=proxy_auth, session=session)
+
+    async def get_invite(self, invite_code: str, with_counts=False) -> Invite:
+        """|coro|
```

Comment:
Codacy found an issue: [1 blank line required between summary line and description (found 0) (D205)](https://app.codacy.com/manual/Yandawl/restcord.py/pullRequest?prid=5910681)